packages/dd-trace/src/appsec/reporter.js

```diff
@@ -88,16 +88,18 @@ function formatHeaderName (name) {
     .toLowerCase()
 }
 
-function reportWafInit (wafVersion, rulesVersion, diagnosticsRules = {}) {
-  metricsQueue.set('_dd.appsec.waf.version', wafVersion)
-
-  metricsQueue.set('_dd.appsec.event_rules.loaded', diagnosticsRules.loaded?.length || 0)
-  metricsQueue.set('_dd.appsec.event_rules.error_count', diagnosticsRules.failed?.length || 0)
-  if (diagnosticsRules.failed?.length) {
-    metricsQueue.set('_dd.appsec.event_rules.errors', JSON.stringify(diagnosticsRules.errors))
+function reportWafInit (wafVersion, rulesVersion, diagnosticsRules = {}, success) {
```

Comment:
uh, having an optional argument not be at the end is weird no ? at this point do this ?

```suggestion
function reportWafInit (wafVersion, rulesVersion, diagnosticsRules = {}, success = true) {
```

Reply:
Yes looks weird, done